lib/src/klio/transforms/io.py

```diff
@@ -360,6 +360,19 @@ class KlioReadFromAvro(beam.io.ReadFromAvro):
     Data from avro is dumped into JSON and assigned to ``KlioMessage.data.
     element``.
 
+    ``KlioReadFromAvro`` is the default read for event input config type avro.
+    However,``KlioReadFromAvro`` can also be called explicity in a pipeline.
```

Comment:
Need a space after "However," - the `KlioReadFromAvro` isn't rendering quite right:
 
![image](https://user-images.githubusercontent.com/1191069/106054535-2c660880-60ba-11eb-9930-32fa3d50e923.png)

It's happening in the same spot of `KlioWriteToAvro` as well.